pkg/operator/controller.go

```diff
@@ -49,17 +49,23 @@ type Controller struct {
 
 	ocSecWorkqueue workqueue.RateLimitingInterface
 
+	cfgMapWorkqueue workqueue.RateLimitingInterface
+
 	crInformer cache.SharedIndexInformer
 	isInformer cache.SharedIndexInformer
 	tInformer  cache.SharedIndexInformer
 
 	ocSecInformer cache.SharedIndexInformer
 
+	cfgMapInformer cache.SharedIndexInformer
```

Comment:
Just a nit, but all of the different kinds of things here could be grouped together without any spaces between the individual components

Reply:
part of next push